pkg/datasource/csvsource/types.go

```diff
@@ -0,0 +1,16 @@
+package csvsource
+
+import (
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+)
+
+type CsvTick struct {
+	Timestamp       int64            `json:"timestamp"`
+	Symbol          string           `json:"symbol"`
+	Side            string           `json:"side"`
```

Comment:
is it possible to apply types.SideType here?

Reply:
certainly